fintual-oss/paraffin-front
# Landing page - first version

### Context
We are developing the platform's landing page. It introduces our product to the users, indicating its main features and inviting them to register on it.

The landing page has been thought with these sections:
- Header
- Problem
- Solution
- FAQs
- Footer

### Changelog
- Simplify index.js.
- Add images file on src/components/images/images.js.
- Create a JS file and a SCSS file for each of the sections.
- Define a variable file on src/styles/variables.scss.
- Include 12 image files.

### QA
Run the docker container of the Rails app.
Run the frontend server.
Go to http://localhost:3001.
You should the new version of the landing page.
Please take a look at grammar and spelling in Spanish, and tell us if any of the texts does not make sense to you.
